### DISCUSSION TITLE
GitHub Foundation Certification Not Free with GitHub Student Pack

### DISCUSSION THREAD

**Discussion by leoferamos** [community/community#143779] [GitHub Education] [answered]
### Hi everyone,

I’ve had my GitHub Student Pack approved for a while now, and I can access all the benefits:
![image](https://github.com/user-attachments/assets/3552abe3-582c-49ee-9c54-ce8351b483dc)
However, when I try to schedule the GitHub Foundation certification, I only see a $49.5 discount
![image](https://github.com/user-attachments/assets/0e410700-78d7-4859-b66f-5ccef056011e)
Has anyone else experienced this issue or know how to resolve it? Is there something else I need to do to claim the free certification?

**Comment by davevad93** [comment, marked as answer]
Hi @leoferamos ,

There was a bug and they had to disable the offer, technically they should have fixed it by October 31 but they haven't done it yet.  
If you want to stay tuned, subscribe to this discussion #138834 , as soon as the free certification is available again they will announce it there.

![1](https://github.com/user-attachments/assets/896bb6b8-dc91-4251-bf40-1619f3ac23dd)
